src/validator.ts

```diff
@@ -12,17 +11,13 @@ export class Validator {
             verbose: true,
             allErrors: true,
             allowUnionTypes: true,
+            validateFormats: false,
             strictTypes: false, // to suppress the missing types defined in the gitlab-ci json schema
+            keywords: ["markdownDescription"],
         });
-        ajv.addKeyword("markdownDescription");
-        addFormats(ajv);
         const validate = ajv.compile(schema);
         const valid = validate(data);
-        assert(valid,
-            chalk`Invalid gitlab-ci configuration! It have failed the json schema validation. Dump the following to the pipeline editor to debug:
-
-${yaml.dump(data)}
-`);
+        assert(valid, chalk`Invalid gitlab-ci configuration! It have failed the json schema validation. Dump the following to the pipeline editor to debug: ${yaml.dump(data)}`);
```

Comment:
I see the final new line have been removed. 

I’m away from computer to test it locally, but does  yaml.dump already have a new line ?